### PR TITLE
(docs) Readme fixes

### PR DESCRIPTION
**Pull request by ferventcoder**
- The docs should closer match style guide.
- Correct misspellings
- Remove incorrect note about composite resources.
- Fix grammar in some places.
- Add note about setting refresh mode.

Supersedes #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+
+This module generates Puppet Types based on DSC resources' MOF (Managed Object Format) schema files.
 
 In this version, the following DSC Resources are already built and ready for use:
-- All base DSC resources found in PowerShell 5. ([WMF 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=46889))
-- All DSC resources found in the [Microsoft PowerShell DSC Resource Kit](https://github.com/powershell/DSCResources).
+- All base DSC resources found in PowerShell 5. ([WMF 5.0][wmf-5.0])
+- All DSC resources found in the [Microsoft PowerShell DSC Resource Kit][DSCResources]
 
 This module is available on the [Puppet Forge](https://forge.puppetlabs.com/puppetlabs/dsc)
```
Comment (jpogran) [inline README.md]: This link doesn't work yet, but that is expected.

Comment (ferventcoder) [inline README.md]: Correct.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
 
-## Windows systems prerequisites
+## Windows system prerequisites
 
- - PowerShell 5 which is included in the [Windows Management Framework 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=46889). PowerShell v5 is currently in limited preview, so the above link may change after official release.
+ - PowerShell 5 which is included in the [Windows Management Framework 5.0][wmf-5.0]. PowerShell v5 is currently in limited preview, so the above link may change after official release.
```
Comment (jpogran) [inline README.md]: Either of these sounds better

PowerShell 5 which is included in the Windows Management Framework 5.0 release

or

PowerShell 5 which is included in Windows Management Framework 5.0.

File: README.md [hunk 3]
```diff
@@ -36,9 +49,7 @@ dsc_windowsfeature {'IIS':
 
 All DSC Resource names and parameters have to be in lowercase, e.g: `dsc_windowsfeature` or `dsc_name`.
 
-## Example
-
-The following example class will install the 'Backery' website.
+### Install website example
```
Comment (jpogran) [inline README.md]: Is 'Website Installation Example' better?

File: README.md [hunk 4]
```diff
@@ -110,10 +121,10 @@ All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metapa
 
 ## Limitations
 
-- DSC Composite Resources are not yet fully supported.
+- DSC Composite Resources are not supported.
 - PSObjects like 'PSCredential' as parameters value not yet supported.
```
Comment (jpogran) [inline README.md]: should be 'are not yet supported'

File: README.md [hunk 5]
```diff
@@ -110,10 +121,10 @@ All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metapa
...
 
-## Known Issues
+### Known Issues
 
 When installing the module on Windows you may run into an issue regarding long file names (LFN), due to the long paths of generated schema files. If you install your module on a Linux master and then use plugin sync, you will not likely see this issue, but if you are attempting to install the module on a Windows machine, you may run into an error that looks similar to the following during `puppet module install puppetlabs-dsc`:
```
Comment (jpogran) [inline README.md]: Commas, rearranged some parts, run on sentences.

When installing the module on Windows you may run into an issue regarding long file names (LFN) due to the long paths of the generated schema files. If you install your module on a Linux master and then use plugin sync you will  likely not see this issue. If you are attempting to install the module on a Windows machine using `puppet module install puppetlabs-dsc` you may run into an error that looks similar to the following:

File: README.md (hunk 4 above)
Comment (ferventcoder) [inline README.md]: I prefer not supported. We don't know if we will, so it's best not to even state yet.

File: README.md (hunk 5 above)
Comment (ferventcoder) [inline README.md]: Thanks, I wasn't in love with the wording there.

File: README.md (hunk 3 above)
Comment (ferventcoder) [inline README.md]: :+1:

File: README.md (hunk 2 above)
Comment (ferventcoder) [inline README.md]: Fixed.

**Comment by ferventcoder** [comment]
Another one bites the dust! `\o/`

**Pull request merged**